.changelog/1829.feature.md

```diff
@@ -0,0 +1,3 @@
+Add gRPC Sentry Nodes support.
+
+TODO: description
```

Comment:
actually this should probably be `1829.breaking.md` due to the changes to the committe address in registry

Reply:
:+1: